app/frontend/src/components/forms/manage/ManageVersions.vue

```diff
@@ -0,0 +1,208 @@
+<template>
+  <div>
+    <BaseInfoCard class="my-4">
+      When you create a form, it becomes a Draft. Once you publish this form,
+      you are no longer able to edit it, instead you can create a new version
+      and edit from there. A new version of the form will carry the existing
+      version that you created. If you unpublish the form, the submitter won't
+      be able to access the form.
+    </BaseInfoCard>
+
+    <CurrentDraft v-if="hasDraft" class="my-8" />
+
+    <strong>History</strong>
+    <v-data-table
+      class="submissions-table"
+      :headers="headers"
+      :items="versionList"
+    >
+      <template #no-data>
+        No versions yet. Publish a draft to release a form version.
+      </template>
+
+      <!-- Version  -->
+      <template #[`item.version`]="{ item }">
+        <router-link
+          :to="{
+            name: 'FormPreview',
+            query: { f: item.formId, v: item.id },
+          }"
+          class="mx-5"
+          target="_blank"
+        >
+          <v-tooltip bottom>
+            <template #activator="{ on, attrs }">
+              <span v-bind="attrs" v-on="on">Version {{ item.version }}</span>
+            </template>
+            <span> Click to preview <v-icon>open_in_new</v-icon></span>
+          </v-tooltip>
+        </router-link>
+      </template>
+
+      <!-- Published date  -->
+      <template #[`item.createdAt`]="{ item }">
+        {{ item.createdAt | formatDateLong }}
+        - {{ item.createdBy }}
+      </template>
+
+      <!-- Create new version  -->
+      <template #[`item.create`]="{ item }">
+        <v-tooltip bottom>
+          <template #activator="{ on, attrs }">
+            <v-btn
+              color="primary"
+              text
+              small
+              @click="createVersion(item.formId, item.id)"
+              v-bind="attrs"
+              v-on="on"
+            >
+              <v-icon>add_circle</v-icon>
+            </v-btn>
+          </template>
+          <span>
+            Use version {{ item.version }} as the base for a new version
+          </span>
+        </v-tooltip>
+      </template>
+
+      <!-- Export Schema  -->
+      <template #[`item.export`]="{ item }">
+        <v-tooltip bottom>
+          <template #activator="{ on, attrs }">
+            <v-btn
+              color="primary"
+              text
+              small
+              @click="exportSchema(item.id)"
+              v-bind="attrs"
+              v-on="on"
+            >
+              <v-icon>get_app</v-icon>
+            </v-btn>
+          </template>
+          <span>Download version {{ item.version }} to a file</span>
```

Comment:
"Download version ## schema" instead?

Reply:
I don't think we use the word schema to the users anywhere else but could be wrong. Will see what Mina wants